lib/telephony/pinpoint/sms_sender.rb

```diff
@@ -13,7 +13,7 @@ class SmsSender
         'UNKNOWN_FAILURE' => UnknownFailureError,
       }.freeze
 
-      # rubocop:disable Metrics/MethodLength, Metrics/AbcSize, Metrics/BlockLength
+      # rubocop:disable Metrics/BlockLength
```

Comment:
There's a matching `# rubocop:enable ...` below. Should it be updated to the new set of enabled rules?